index.html

```diff
@@ -0,0 +1,80 @@
+            fill="#6070FF" />
+        </svg>
+      </li>
+      </ul>
+    </nav>
+
+    <header class="hero-wrapper container">
+      <h1>I’m Sidney
+      Glad to see you!</h1>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <h2>let's connect</h2>
+      <ul class="social">
+        <li>
+          <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path
+              d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
+              fill="#505F79" />
+          </svg>
+        </li>
+
+        <li>
+          <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path
+              d="M13.6448 3.03455C10.1561 3.03455 7.28906 5.86213 7.28906 9.37937V19.3104C7.28906 19.6897 7.59994 20.0001 7.97991 20.0001H11.5723C11.9523 20.0001 12.2632 19.6897 12.2632 19.3104V9.65524C12.2632 8.89661 12.8849 8.27592 13.6448 8.27592C14.4048 8.27592 15.0265 8.89661 15.0265 9.65524V19.3104C15.0265 19.6897 15.3374 20.0001 15.7174 20.0001H19.3098C19.6898 20.0001 20.0006 19.6897 20.0006 19.3104V9.37937C20.0006 5.89662 17.1682 3.03455 13.6448 3.03455Z"
+              fill="#505F79" />
+            <path
+              d="M2.48705 0C1.10535 0 0 1.10345 0 2.48276C0 3.86207 1.10535 4.96552 2.48705 4.96552C3.86874 4.96552 4.97409 3.86207 4.97409 2.48276C4.97409 1.10345 3.86874 0 2.48705 0Z"
+              fill="#505F79" />
+            <path
+              d="M4.28325 6.27588H0.690846C0.310881 6.27588 0 6.58622 0 6.96553V19.3104C0 19.6897 0.310881 20 0.690846 20H4.28325C4.66321 20 4.97409 19.6897 4.97409 19.3104V6.96553C4.97409 6.58622 4.66321 6.27588 4.28325 6.27588Z"
+              fill="#505F79" />
+          </svg>
+        </li>
+        <li>
+          <svg width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">

```

Comment:
This social media button should be linked to the proper website later in the project, so make sure they are inside the `<a>` tag link.

Reply:
got you!